### DISCUSSION TITLE
Unable to Push my content to Supabase Database.

### DISCUSSION THREAD

**Discussion by Pokah1** [supabase/supabase#28352] [Questions]
Been working on a project that has to do with markdown editor and save the content to Supabase post table.

I keep getting 505 error or Auth missing session error.

**Comment by GaryAustin1** [comment]
You need to give more info on your call to Supabase.   There is no "push" call so not clear what that means.

**Reply by Pokah1** [reply]
Okay. 
I'm just a newbie starting working with Supabase.
I am trying to build a project where I can write , edit, delete contents or posts using Markdown tool.

I have been able to set up the authentication on server side. But this particular content feature I am working on requires me working on client side and saving the content to table on database .
Once I start introducing useState and useEffect with
Import Supabase from utilis/Supabase/client.

I get an error of user not authenticated,
Auth missing key.
However, once I try to do it on server side like 
Import (createClient) from utilis/Supabase/server,
I'll get the user .

I could share my different codebase here for future assistance please
